src/contrib/mlir/backend/pass/affine_lowerer.cpp

```diff
@@ -1410,6 +1476,30 @@ namespace
         }
         NGRAPH_UNREACHABLE("Unsupported type");
     }
+
+    ValueHandle createOneConstant(mlir::Type type)
+    {
+        if (auto floatTy = type.dyn_cast<FloatType>())
+        {
+            if (floatTy.isF32())
+            {
+                return intrinsics::constant_float(llvm::APFloat(1.0f), floatTy);
```

Comment:
Could you please check if there is something to create one constant given a type. There has to be something in the builder or constant op.